Parse/PFObject.m

```diff
@@ -1634,7 +1634,9 @@ + (instancetype)objectWithClassName:(NSString *)className {
 
 + (instancetype)objectWithClassName:(NSString *)className dictionary:(NSDictionary *)dictionary {
     PFObject *object = [self objectWithClassName:className];
-    [object _mergeAfterSaveWithResult:dictionary decoder:[PFDecoder objectDecoder]];
+    [dictionary enumerateKeysAndObjectsUsingBlock:^(id key, id obj, BOOL *stop) {
+        object[key] = [[PFDecoder objectDecoder] decodeObject:obj];
```

Comment:
Nit: You can save some method invocations by using a single local variable for storing the decoder. Feel free to leave as is, as it's super nitpicking.